output/notify/notify.go

```diff
@@ -114,7 +114,7 @@ func (n *NotifyOutput) Start() error {
 						"New Transaction!\nBlockNumber: %d, SlotNumber: %d\nInputs: %d, Outputs: %d\nFee: %d\nHash: %s",
 						tc.BlockNumber,
 						tc.SlotNumber,
-						len(te.Inputs),
+						len(te.Inputs)+len(te.ReferenceInputs),
```

Comment:
I'm not sure that we'd want to count reference inputs with the actual TX inputs like this